test/battle/move_effect/evasion_up_2.c

```diff
@@ -0,0 +1,28 @@
+#include "global.h"
+#include "test/battle.h"
+
+// There's no move with EFFECT_EVASION_UP_2 effect. Below is a theorical test.
```

Comment:
```suggestion
// There's no move with EFFECT_EVASION_UP_2 effect. Below is a theoretical test.
```
I'm not sure if there's a point in these though, we barely cover all non-theoretical cases and doing this sets a precedent that may make people expect us to write tests for their hypothetical new effects, too

Reply:
Should we remove this unused move effect then? I believe it's from vanilla

Reply:
I think it's highly unlikely to be introduced in (vanilla) Pokémon at any point, so fine by me

Reply:
Can we keep it? Since it is already there, there is no harm in keeping it. It might also be an effect users used for their custom moves. 

Though I think we should not introduce custom effects

Reply:
Did some digging and indeed `EFFECT_EVASION_UP_2` is from vanilla.

Reply:
https://github.com/pret/pokeemerald/blob/50d325f081a161f4a223d999497d7a65bd896194/include/constants/battle_move_effects.h#L60